services/grpc/internal/transformations/graph_objects.go

```diff
@@ -0,0 +1,36 @@
+package transformations
+
+type Node struct {
+	NodeId string `db:"nodeId"`
+	Doc string `db:"doc"`
+
+	EdgesOut string[] `db:"edgesOut"` // IDs of edges whose destination is this node
+	EdgesIn string[] `db:"edgesIn"`   // ID of edges whose source is this node
+
+	// Metadata
+	Date string `db:"date"`
+	Author string `db:"author"`
+}
+
+type Edge struct {
+	EdgeId string `db:"edgeId"`
+	SourceDoc string `db:"sourceDoc"`
+	TargetDoc string `db:"targetDoc`
```

Comment:
Could an Edge be used / "belong" to multiple Scopes? I generally like the idea of keeping it as an option.

Reply:
TODO: Remove Docs.